src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -195,7 +195,9 @@ class UserService(protected val userInfo: UserInfo, dataSource: SlickDataSource,
     getUserStatus(RawlsUserRef(RawlsUserSubjectId(userInfo.userSubjectId)))
   }
 
-  def getUserStatus(userRef: RawlsUserRef): Future[PerRequestMessage] = withUser(userRef) { user =>
+  private def loadUser(userRef: RawlsUserRef): Future[RawlsUser] = dataSource.inTransaction { dataAccess => withUser(userRef, dataAccess)(DBIO.successful) }
```

Comment:
my suspicion is that this'll want to move to UserWiths too, but i'm willing to postpone that until it's needed